server/src/index.ts

```diff
@@ -1,20 +1,32 @@
-import { Hono } from 'hono';
-import { cors } from 'hono/cors';
-import v1App from './router/v1/v1.router';
+import { Hono } from "hono";
+import { cors } from "hono/cors";
+import v1App from "./router/v1/v1.router";
 
 const app = new Hono();
 
-app.use('/*', cors());
-app.get('/', c =>
-	c.json({
-		status: 'ok',
-	}),
+app.use("/*", cors());
+app.get("/", (c) =>
+  c.json(
+    {
+      status: "ok",
+    },
+    200
+  )
+);
+
+app.get("/test", (c) =>
```

Comment:
we already have the healthcheck `/` endpoint
for LB healthcheck in the future

Reply:
forgot to remove it